src/app/api/schemas.py

```diff
@@ -160,10 +160,10 @@ class AlertIn(BaseModel):
     device_id: int = Field(..., gt=0)
     event_id: int = Field(..., gt=0)
     media_id: int = Field(None, gt=0)
-    lat: float = Field(..., gt=-90, lt=90)
-    lon: float = Field(..., gt=-180, lt=180)
+    lat: float = Field(..., gt=-90, lt=90, example=48.654321)
+    lon: float = Field(..., gt=-180, lt=180, example=2.654321)
     type: AlertType = AlertType.start
-    is_acknowledged: bool = False
+    is_acknowledged: bool = Field(False, example=False)
```

Comment:
I think I misspoke earlier : since it's typed as bool, I think the example is not required :thinking:

Reply:
hum, indeed, I will check without example and remove it if necessary